content/_guides/connect.md

```diff
@@ -12,6 +12,8 @@ Additional regional and address-specific hostnames are available:
 
 | ------------------------- | ---------------------- |
 | Default                   | `irc.libera.chat`      |
+| Europe                    | `irc.eu.libera.chat`   |
+| US & Canada               | `irc.us.libera.chat`   |
```

Comment:
Why not North America?

But lgtm with the formatting feel free to merge

Reply:
We've never had a server outside of those countries. We can update the description if it happens.